### PR TITLE
feat: add minimum functionality and UI to seed phrase screen

### LINKED ISSUE #109: Implement seed phrase screen
## Description

To implement the seed phrase screen, the following steps will be taken:

1. Implement a dummy seed phrase screen (without the actual seed phrase words)
2. Implement screen capture blocker
3. Implement seed phrase generator

### PR DESCRIPTION
**Pull request by juanmanso**
## Description

In this PR we added the UI/UX for the screen (still some styling needs to be fixed) where the user can hide and show the seed phrase and also copy it to the clipboard.

_Next steps_
- Adding screen blocking functionality
- Adding seed phrase generation from entropy

## Related links
- Closes #109 

## Demo

| _iOS_ | _Android_ |
| :---: | :---: |
| <video src='https://github.com/dominant-strategies/quaipay/assets/21087992/f32a73c8-6879-49fa-a612-e012e55ee80a' width=400> | <video src='https://github.com/dominant-strategies/quaipay/assets/21087992/cf87a34b-8245-4e01-aea3-59530c022772' width=400> |

### CODE REVIEW

File: src/shared/locales/de/export.json [hunk 1]
```diff
@@ -13,5 +13,14 @@
       }
     },
     "learnMore": "Erfahren Sie mehr über den Wiederherstellungsprozess."
+  },
+  "phrase": {
+    "title": "Schreiben Sie Ihre Seed-Phrase auf",
+    "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus. Se",
+    "screenShotBanner": "Screenshots sind deaktiviert, um den Schutz Ihrer Wiederherstellungs-\nphrase zu gewährleisten",
+    "revealPhrase": "Zeigen Sie Ihre Seed-Phrase an",
```
Comment (obasilakis) [inline src/shared/locales/de/export.json:21]: let's change to: "Seed-Phrase anzeigen"

File: src/shared/locales/de/export.json [hunk 2]
```diff
@@ -13,5 +13,14 @@
...
+    "hidePhrase": "Verstecken Sie Ihre Seed-Phrase",
```
Comment (obasilakis) [inline src/shared/locales/de/export.json:22]: "Seed-Phrase ausblenden"

File: src/main/settings/export/components/SeedPhraseDisplay.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { QuaiPayText } from 'src/shared/components';
+import { useThemedStyle } from 'src/shared/hooks/useThemedStyle';
+import { Theme } from 'src/shared/types';
+
+const placeholderWord = 'Octopus';
+const placeholderSecuredWord = placeholderWord.split('').map(() => '*');
+
+interface ISeedPhraseDisplayProps {
```
Comment (obasilakis) [inline src/main/settings/export/components/SeedPhraseDisplay.tsx:10]: the power of habit? :D

Comment (juanmanso) [reply src/main/settings/export/components/SeedPhraseDisplay.tsx:10]: haha just a snippet that I have configured that generates me a component from scratch with interface and styling. Let me change it, it slipped through 😅

**Review by obasilakis — APPROVED** [review]

**Review by hsyndeniz — APPROVED** [review]

**Pull request merged**